applications/plugins/Geomagic/src/Geomagic/GeomagicEmulator.cpp

```diff
@@ -167,7 +175,11 @@ void GeomagicEmulator::initDevice()
     }
 
     // 2.6- Need to wait several ms for the scheduler to be well launched and retrieving correct device information before updating information on the SOFA side.
-    Sleep(42);
+#ifndef WIN32
```

Comment:
Please use 
` std::this_thread::sleep_for(std::chrono::milliseconds(x)); `
instead of sleep/Sleep and the ifdefs

Reply:
CTime was done because of a lack of time API standard; now that  there is a stdlib for that, I would advocate to remove CTime altogether...

Reply:
I will be interested to see the changes because right now I'm using CTime tick mechanism to be more precise than method sleep_for for example. See my other comment below.